src/containers/project-overview/index.tsx

```diff
@@ -38,6 +38,7 @@ export const ProjectOverview: React.FC<ProjectOverviewContainerProps> = ({ supab
     }
 
     loadProjects();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
question, why don't we just add the dependency in here?

Reply:
Because I don't want react to run this effect more than once when rendering this component and I think this is the only way I can tell it that. When I added in `supabase` I found that the network call for `projects` was being made twice, rather than once when I provided no dependencies (`[]`).

I'm yet to fully delve into how this all works (will read this later https://overreacted.io/a-complete-guide-to-useeffect/) but I think it's because if the object (being passed in as a dependency) changes at all, then the whole function is run again.

<img width="667" alt="image" src="https://github.com/Aid-Pioneers/shipment-aid-tracker/assets/1821099/80916725-63ce-4c3d-b952-eda5d96bc9a6">


I have no idea what the inner workings of the `supabase` client are, but it seems plausible to me that something in it is changing a lot and it will trigger this function each time.

Reply:
@killerdaisies found it.

It is running twice (in dev) because we are running in `StrictMode` 🤦 . So I think it should be fine to add in `supabase` as a dependency. Will do it now...

```typescript
<React.StrictMode>
  <App />
</React.StrictMode>
```